plugins/snowflake/dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -0,0 +1,85 @@
+
+{% materialization incremental, adapter='snowflake' -%}
+
+  {%- set unique_key = config.get('unique_key') -%}
+  {%- set sql_where = config.get('sql_where') -%}
+
+  {%- set non_destructive_mode = (flags.NON_DESTRUCTIVE == True) -%}
+  {%- set full_refresh_mode = (flags.FULL_REFRESH == True) -%}
+
+  {%- set identifier = model['alias'] -%}
+
+  {%- set old_relation = adapter.get_relation(database=database, schema=schema, identifier=identifier) -%}
+
+  {%- set target_relation = api.Relation.create(database=database, identifier=identifier, schema=schema, type='table') -%}
+
+  {%- set exists_as_table = (old_relation is not none and old_relation.is_table) -%}
+  {%- set exists_not_as_table = (old_relation is not none and not old_relation.is_table) -%}
+
+
+  {%- set force_create_or_replace = full_refresh_mode -%}
+
+
+  -- setup
+
+  {% set source_sql -%}
```

Comment:
We're going to drop the `sql_where` config in 0.14.0 (Wilt Chamberlain) -- you can just do 
```
{% set source_sql = sql %}
```
here :)